src/PomodoroTimerPlugin.js

```diff
@@ -17,6 +22,15 @@ const PomodoroTimerPlugin = {
         })
     },
 
+    triggerNotification: function(){
+        const notify = new CustomEvent("notification", {
```

Comment:
I think we should namespace this to something more specific than "notification" - only because lots of other plugins, etc. will use it.  Maybe pomodoro-timer-notification - longer is better/more specific in this case to avoid conflicts with other plugins/libraries